frontmacs-pkg.el

```diff
@@ -23,6 +23,11 @@
     (emmet-mode "1.0.8")
     (smartparens "1.9.0")
     (web-mode "14.1")
+    (diminish "0.45")
+    (which-key "20170315.1055")
+    (volatile-highlights "20160611.1855")
```

Comment:
Let's pin these back to their stable versions, now that stable.melpa.org is back up.